File: algorithms/q9/solution.cpp
#include<iostream>
#include<vector>
using namespace std;

int ocurrence(vector<int> arr,int value){
    int repetition = 0;
    for(int i = 0;i<arr.size();i++){
        if(arr[i] == value){
            repetition++;
        }
    }
    return repetition;
}

int main(){
    vector<int> arr = {1,2,4,4,5,3,2,4,2,1};

    int x;
    cout << "Enter the value to be searched: " << endl;
    cin >> x;

    cout << "Number of times value repeated is: " << ocurrence(arr,x) << endl;

    return 0;
}

File: algorithms/q9/solution.py
def ocurrence(arr,value):
    repetition = 0
    for i in range(len(arr)-1):
        if(arr[i] == value):
            repetition+=1
    return repetition


arr = [1,2,32,1,1,3,4,45,0]
x = int(input("Enter the value: "))
print("Number of times value being repeated is: ",ocurrence(arr,x))
